front/src/modules/simulationResult/components/SpaceTimeChart/ManchetteMenuButton.tsx

```diff
@@ -0,0 +1,75 @@
+import { useEffect, useMemo, useRef, useState } from 'react';
+
+import { Eye, KebabHorizontal } from '@osrd-project/ui-icons';
+import cx from 'classnames';
+import { useTranslation } from 'react-i18next';
+
+import type { OSRDMenuItem } from 'common/OSRDMenu';
+import OSRDMenu from 'common/OSRDMenu';
+import useModalFocusTrap from 'utils/hooks/useModalFocusTrap';
+
+const ManchetteMenuButton = () => {
+  const { t } = useTranslation('simulation');
+  const [isMenuOpen, setIsMenuOpen] = useState(false);
+
+  const menuRef = useRef<HTMLDivElement>(null);
+  const menuButtonRef = useRef<HTMLButtonElement>(null);
+
+  const closeMenu = () => setIsMenuOpen(false);
+
+  const menuItems: OSRDMenuItem[] = useMemo(
+    () => [
+      {
+        title: t('manchetteSettings.waypointsVisibility'),
+        icon: <Eye />,
+        onClick: () => {
+          closeMenu(); // TODO : in #8628, change this to open the waypoints modal
+        },
+      },
+    ],
+    [t]
+  );
+
+  useModalFocusTrap(menuRef, closeMenu);
+
```

Comment:
I see that we already have a custom hook to handle the click ouside. What do you think to refactor to allow to use it in this case too ? Can be done in another PR

Reply:
at least a TODO should be enough

Reply:
Oh yes, good point. It will indeed need a refacto as I need to pass 2 refs instead of one.
I added a TODO.